bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties

```diff
@@ -14,5 +14,6 @@ offline.conf-error-auth = Autenticazione fallita. Verificare la password nella c
 offline.comm-error-disconnected = Disconnesso dal gateway.
 offline.comm-error-timeout = Connessione al gateway non riuscita in tempo.
 offline.comm-error-connection = Impossibile connettersi al gateway.
+offline.comm-error-state = Impossibile aggiornare lo stato. 
```

Comment:
Please do not update the Italian translation in this PR; do it later in Crowdin.

Reply:
removed change